### DISCUSSION TITLE
No more videos, please!

### DISCUSSION THREAD

**Discussion by RenanLazarotto** [dailydotdev/daily#1069] [Feature Requests]
Please add an option to filter out articles that are, in fact, videos. While it may be good in some scenarios, I find them time-consuming when compared to just reading an article, more often than not the video links to some article and/or repository which has the code shown on video, whereas most text articles just include the code in its body.

Suggestion:
![image](https://github.com/dailydotdev/daily/assets/1446403/4ee260f2-af05-43a9-bfe5-131cd79c5670)

**Comment by nimrodkra** [comment]
@RenanLazarotto, it exists but not on this menu. On your feed, click Feed settings -> Content types -> Disable the Video checkbox. 

![Screenshot 2024-02-03 at 15 17 59](https://github.com/dailydotdev/daily/assets/41470823/07458cce-c19e-45e9-8e59-a158e1de1bb6)

**Reply by RenanLazarotto** [reply]
Oh good, I didn't knew about this. Thanks you! I guess I should take a deeper look at the available options next time...

**Reply by marvinpoo** [reply]
Thank you. I was about to go insane. The last 2 days I have been bombarded with videos and not a single written article. Thank you so much for this life-time saving feature!

**Reply by marvinpoo** [reply]
Actually, this seems to be still not working correctly. 
I get video posts on a daily basis, even though I have deactivated the option.

![image](https://github.com/user-attachments/assets/d91a720e-5372-4eb6-a325-2ce3ac05b485)
![image](https://github.com/user-attachments/assets/4e460054-4776-44c0-ab36-dbb783d7fcc1)

**Reply by idoshamun** [reply]
You're right. It happens because of shared posts, if someone shares a YT link in a squad it ignores this filter. We need to fix it
